templates/common/powervs/units/powervs-hostname.service.yaml

```diff
@@ -11,8 +11,8 @@ contents: |
   Before=node-valid-hostname.service
```

Comment:
You have to add:
```
Wants=afterburn.service
After=afterburn.service
```
here, or afterburn won't run at all (Wants), and won't run in the correct order (After).